.github/workflows/sc-docker-client-server-bldr.yml

```diff
@@ -98,6 +98,6 @@ jobs:
       run: ./exec.sh --no-tty -i client pytest --testbed=saivs_client_server -v -k "test_l2_basic"
     - name: Run unit tests
       run: ./exec.sh --no-tty -i client pytest --testbed=saivs_client_server -v -k \
-           "test_acl_ut or test_bridge_ut or test_switch_ut or test_vrf_ut or test_port_ut.py"
+           "test_acl_ut or test_bridge_ut or (test_switch_ut and not sai_map_list_t) or test_vrf_ut or test_port_ut.py"
```

Comment:
Looks like there is a bug in `saivs` which cases the failure as follows:
```
Apr  4 07:25:30 b2cb2e0f8edf saiserver: :- meta_genetic_validation_list: SAI_SWITCH_ATTR_PFC_TC_DLD_INTERVAL:SAI_ATTR_VALUE_TYPE_MAP_LIST when count is zero, list must be NULL
```
So, disabling for now...